package.json

```diff
@@ -23,6 +23,7 @@
 		"@babel/preset-env": "^7.18.6",
 		"@babel/preset-react": "^7.18.6",
 		"@babel/preset-typescript": "^7.18.6",
+		"@svgr/cli": "^6.3.1",
```

Comment:
**nitpick**: it seems that this dependency is unused..

Reply:
It is, by the new script "illustration" :)